ssz-rs/src/lib.rs

```diff
@@ -9,21 +14,19 @@ mod merkleization;
 mod ser;
 #[cfg(feature = "serde")]
 mod serde_test;
+mod std;
 mod uint;
 mod union;
 mod vector;
 
-use crate::list::Error as ListError;
-use crate::vector::Error as VectorError;
+pub use crate::std::*;
 pub use bitlist::Bitlist;
 pub use bitvector::Bitvector;
 pub use de::{Deserialize, DeserializeError};
 pub use list::List;
 pub use merkleization::{Context as MerkleizationContext, MerkleizationError, Merkleized, Node};
 pub use ser::{Serialize, SerializeError};
-use thiserror::Error;
 pub use uint::U256;
-pub use vector::Vector;
```

Comment:
why do we drop the export of `Vector` here?

Reply:
Added back in https://github.com/ralexstokes/ssz-rs/pull/25/commits/02961035ee736414a965e9f8fefceae01e1c98a4.